content/styleguide/styleguide.rst

```diff
@@ -0,0 +1,623 @@
+
+.. code-block:: rst
+
+  The enumerator uses his device.
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  The enumerator uses their device.
+
+
+Unfortunately, *they/them/their* is not a perfect solution. Since it is conventionally used as a plural pronoun, it can cause confusion.
+
+Therefore, avoid the use of personal pronouns whenever possible. They are often out of place in technical writing anyway. Rewriting passages to avoid personal pronouns often makes the writing more clear.
+
+.. rubric:: Correct
+
+.. code-block:: rst
+
+  When using Collect, first the enumerator opens the app on their device. Then they complete the survey.
+  
+.. rubric:: Better
+
+.. code-block:: rst
+
+  To use Collect:
+  
+  - open the app
+  - complete the survey
+
+.. _same:  
+  
+"Same"
+~~~~~~~~~
+
+*Same*, when used as an impersonal pronoun, is non-standard in Modern American English. It should be avoided.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  ODK Collect is an Android app. The same can be used for...
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  ODK Collect is an Android app. It can be used for...
+
+.. rubric:: Right
+
+.. code-block:: rst
+  
+  ODK Collect is an Android app that is used to...
+  
+
+.. _titles-style-guide:  
+  
+Titles 
+------------
+
+.. _title-casing:
+
+Title case and sentence case
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Document titles should be in ``Title Case`` -- that is, all meaningful words are to be capitalized.
+
+Section titles should use ``Sentence case`` -- that is, only the first word should be capitalized, along with any proper nouns or other words usually capitalized in a sentence.
+
+.. _title-verb-forms:
+
+Verb forms
+-----------
+
+If a document or section describes a procedure that someone might do, use a verb ending in *-ing*. (That is, a `gerund <https://en.wikipedia.org/wiki/Gerund>`_.) Do not use the "How to..." construction.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  How to install ODK Collect
+  --------------------------
+    
+.. rubric:: Right
+
+.. code-block:: rst
+
+  Installing ODK Collect
+  ----------------------
+    
+If section title is a directive to do something (for example, as a step in a procedure), use an imperative. 
+
+.. code-block:: rst
+
+  Installing ODK Aggregate
+  ------------------------
+  
+  Download ODK Aggregate
+  ~~~~~~~~~~~~~~~~~~~~~~
+
+  Section content here.
+
+  
+.. _section-label-style-guide:  
+  
+Section labels
+~~~~~~~~~~~~~~~~
+
+Section titles should almost always be preceded by labels.
+
+The only exception is very short subsections that repeat --- like the **Right** and **Wrong** titles in this document.
+
+In these cases, you may want to use the :rst:dir:`rubric` directive.
+    
+.. _other-title-considerations:
+      
+Other titling considerations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Do not put step numbers in section titles.
+- Readers skim. Section titles should be clear and provide information.
+
+  
+.. _writing-about-code:
```

Comment:
I don't think we will have any code in our documentation. I think we can delete everything from here down.